101-webapp-including-insights/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webappName": {
+      "value": "GEN-UNIQUE"
+    },
+    "appInsightsName": {
+      "value": "GEN-UNIQUE"
+    },
+    "hostingPlanName": {
+      "value": "GEN-UNIQUE"
+    },
+    "skuName": {
+      "value": "S1"
+    },
+    "owner": {
```

Comment:
remove this and use a defaultValue in the template

Reply:
done